examples/tasks/task-mem.json

```diff
@@ -19,7 +19,7 @@
       },
       "publish": [
         {
-          "plugin_name": "file",
+          "plugin_name": "mock-file",
```

Comment:
That's because plugin being loaded in the test is "snap-plugin-publisher-mock-file" (examples/tasks/mock-meminfo.sh:30). Without this change tests were failing.